### DISCUSSION TITLE
Convert MHTML to SingleFile HTML

### DISCUSSION THREAD

**Discussion by marco-eckstein** [gildas-lormeau/SingleFile#1182] [Q&A]
I have a large collection of MHTML files. What is the best way to convert them to SingleFile HTML files?

**Comment by roma-marshall** [comment]
hello @marco-eckstein, check it out: https://products.groupdocs.app/conversion/mhtml-to-html

**Reply by marco-eckstein** [reply]
Thanks. It would be just what I need if it would work better. The layout gets completely messed up in the conversion. But it's something.
